libs/common/Memory/LimitedFixedBufferPool.cs

```diff
@@ -61,8 +69,8 @@ public void Return(PoolEntry buffer)
                         Interlocked.Decrement(ref pool[level].size);
```

Comment:
This needs commenting. My first thought on seeing .size was to wonder why we can't just use the queue.Count method. Now it looks like .size is total number of allocations? The PoolLevel field name comments are uninformative.

Reply:
I am not sure what the comment is here? LFBP implementation has not change. I just added Purge functionality and GetStats option. There might be a need to redesign the buffer pool but should part of another PR.